src/app/component/core/header-new/header-new.component.html

```diff
@@ -1,24 +1,27 @@
-<div class="main">
+<div [ngClass]="onToggleBurgerMenu ? 'break-shadow main' : 'main'">
   <div class="main-container">
-    <div class="header">
+    <div class="header" >
       <div class="logo">
         <a href="#">
           <picture>
           <source media="(max-width: 576px)"
                   srcset="assets/img/logo_S.png">
-          <img src="assets/img/logo_M.png" alt="logo">
+             <img src="assets/img/logo_M.png" alt="logo">
```

Comment:
Please write attributes in separate rows

Reply:
+